setup.py

```diff
@@ -899,7 +899,7 @@ def get_exts_for(name):
             CMakeExtension("ddtrace.appsec._iast._taint_tracking._native", source_dir=IAST_DIR, optional=False)
         )
 
-    if CURRENT_OS in ("Linux", "Darwin") and is_64_bit_python():
+    if CURRENT_OS in ("Linux", "Darwin") and is_64_bit_python() and sys.version_info < (3, 14):
         native_features.append("crashtracker")
```

Comment:
```suggestion
    if CURRENT_OS in ("Linux", "Darwin") and is_64_bit_python():
        if sys.version_info < (3, 14)
            native_features.append("crashtracker")
```

@taegyunkim like this?